### DISCUSSION TITLE
Which argocd version can i use for kubernetes 1.18.20?

### DISCUSSION THREAD

**Discussion by YifuYan** [argoproj/argo-cd#12941] [Q&A]
My product enviroment is run with kubernetes v1.18. I looked suitable version for stable argocd's doc, but i have got "Argo CD 2.3.5 on Kubernetes 1.22"... 
I tried to use stable version with kubernetes v1.18, but I got error with helm.

Anyone who knows that whic argocd version can install on the kubernetes v1.18.20? 
Thank you for your informations.

**Comment by kostis-codefresh** [comment]
Kubernetes 1.18 is not even supported by Kubernetes itself. So it makes sense that it is not supported by Argo CD either https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/#supported-versions
